### PR TITLE
Rename and replace all reference to scan upon demand in application app (ex-SCUD API)

### PR DESCRIPTION
**Pull request by YiannisPapastergiou**
GCI-1464

### CODE REVIEW

File: routes.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-app_name: scan-upon-demand.orders.api.ch.gov.uk
+app_name: missing-image-delivery.orders.api.ch.gov.uk
 group: api
 weight: 991
 routes:
-   1: ^/orderable/scans-upon-demand.*
+   1: ^/orderable/missing-image-delivery.*
```
Comment (nabeelamirch) [inline routes.yaml:5]: This should be /orderable/missing-image-deliveries

File: README.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ API handling CRUD operations on CH Scan Upon Demand Ordering Service
 ### Environment Variables
 Name | Description | Mandatory | Location
 --- | --- | --- | ---
-SCAN_UPON_DEMAND_ORDERS_API_PORT | Port this application runs on when deployed. | ✓ | start.sh
+MISSIG_IMAGE_DELIVERY_ORDERS_API_PORT | Port this application runs on when deployed. | ✓ | start.sh
```
Comment (nabeelamirch) [inline README.md:18]: This should be missing not missig

File: routes.yaml (hunk 1 above)
Comment (YiannisPapastergiou) [reply routes.yaml]: done

File: README.md (hunk 2 above)
Comment (YiannisPapastergiou) [reply README.md]: done

File: src/main/java/uk/gov/companieshouse/missingimagedelivery/orders/api/model/MissingImageDeliveryItem.java [hunk 3]
```diff
@@ -7,11 +7,11 @@
 import java.util.List;
 
 /**
- * An instance of this represents a scan upon demand item.
+ * An instance of this represents a missing image delivery item.
  */
 
 @Document(collection = "scan_upon_demands")
```
Comment (hwilliams1) [inline src/main/java/uk/gov/companieshouse/missingimagedelivery/orders/api/model/MissingImageDeliveryItem.java:13]: The collection should be missing_image_deliveries I think?

**Review by hwilliams1 — CHANGES_REQUESTED** [review]
Some files have tabs instead of spaces, there is a reference to scan-upon-demand in the pom.xml, we are using a variable named scanUponDemandItem and the collection name should be changed to be missing_image_deliveries

**Review by nabeelamirch — APPROVED** [review]

**Review by hwilliams1 — APPROVED** [review]

**Pull request merged**